vscode-lean4/src/config.ts

```diff
@@ -201,8 +201,8 @@ export function getTestFolder() : string {
     return typeof(process.env.TEST_FOLDER) === 'string' ? process.env.TEST_FOLDER : '';
 }
 
-export function getTestVersion(): string | undefined {
-    return typeof(process.env.TEST_VERSION) === 'string' ? process.env.TEST_VERSION : undefined;
+export function getDefaultLeanVersion(): string {
+    return typeof(process.env.TEST_VERSION) === 'string' ? process.env.TEST_VERSION : 'leanprover/lean4:nightly';
```

Comment:
Let's also rename the environment var.

Reply:
Oh, yeah, I see what you mean now, good idea, done. See 2f438a787d770623b749f840e0e2e3400fd56094